lib/models/channel_account.ex

```diff
@@ -16,15 +16,23 @@ defmodule ExMicrosoftBot.Models.ChannelAccount do
   """
   @spec parse(map) :: {:ok, ExMicrosoftBot.Models.ChannelAccount.t}
   def parse(param) when is_map(param) do
-    {:ok, Poison.Decode.decode(param, as: decoding_map())}
+    Poison.Decode.decode(param, as: decoding_map())
+  end
+
+  @doc """
+  Decode a list of maps into a list of `ExMicrosoftBot.Models.ChannelAccount`
+  """
+  @spec parse(list) :: {:ok, [ExMicrosoftBot.Models.ChannelAccount.t]}
+  def parse(param) when is_list(param) do
+    Poison.Decode.decode(param, as: [decoding_map()])
   end
 
   @doc """
   Decode a string into `ExMicrosoftBot.Models.ChannelAccount`
   """
   @spec parse(String.t) :: ExMicrosoftBot.Models.ChannelAccount.t
   def parse(param) when is_binary(param) do
-    Poison.decode!(param, as: decoding_map())
+    parse Poison.decode!(param)
```

Comment:
This changes the type as currently it would return ChannelAccount when parsed but with this change it will return {:ok, ChannelAccount}